site/assets/scss/_component-examples.scss

```
@@ -386,7 +388,9 @@
 }
 
 .highlight-toolbar {
-  background-color: $gray-100; // Boosted mod: change background color instead of using .bg-light
-  border: solid $gray-400; // Boosted mod: change border color
+  background-color: $gray-100; // Boosted mod: instead of `var(--bs-tertiary-bg)`
```

I think it doesn't change much on this and could help us with the dark mode later, but as you prefer this one
```suggestion
  background-color: var(--bs-tertiary-bg);
```